examples/keras_recipes/reproducibility_recipes.py

```diff
@@ -151,8 +151,10 @@ def train_model(train_data: tf.data.Dataset, test_data: tf.data.Dataset) -> dict
     )
 
     model.compile(
-        optimizer="adam", loss="sparse_categorical_crossentropy", metrics=["accuracy"]
+        optimizer="adam", loss="sparse_categorical_crossentropy", metrics=["accuracy"], jit_compile = False
```

Comment:
Remove spaces around = and keep line under 80 chars

Reply:
Done! :)